cmd/root.go

```diff
@@ -20,26 +21,35 @@ var rootCmd = &cobra.Command{
 
 		token, err := cmd.Flags().GetString("bot-token")
 		if err != nil || token == "" {
-			log.Error().Err(err).Msg("error in token")
-			return
+			log.Fatal().Err(err).Msg("error in token")
 		}
 
 		time, err := cmd.Flags().GetString("time")
 		if err != nil || time == "" {
-			log.Error().Err(err).Msg("error in time")
-			return
+			log.Fatal().Err(err).Msg("error in time")
 		}
 
 		timezone, err := cmd.Flags().GetString("timezone")
 		if err != nil || timezone == "" {
-			log.Error().Err(err).Msg("error in timezone")
-			return
+			log.Fatal().Err(err).Msg("error in timezone")
 		}
 
-		bot, err := internal.NewBot(token, time, timezone)
+		db, err := cmd.Flags().GetString("database")
 		if err != nil {
-			log.Error().Err(err).Msg("failed to create bot")
-			return
+			log.Fatal().Err(err).Msg("error in database")
+		}
+
+		if db == "" {
+			defaultPath, err := os.Getwd()
+			if err != nil {
+				log.Fatal().Err(err).Send()
+			}
+			db = filepath.Join(defaultPath, "db")
+		}
```

Comment:
this will cause the db to be created at the current directory of the bot which can change depending on where the user executed it. Maybe we should change it to a default path?